example/package.json

```diff
@@ -12,7 +12,7 @@
     "@react-native-community/clipboard": "^1.2.2",
     "react": "16.11.0",
     "react-native": "0.62.2",
-    "react-native-animated-code-input": "file:.."
+    "@brainsbeards/react-native-animated-code-input": "1.0.0"
```

Comment:
let's open up this to "^1.0.0" so we don't have to update it on every patch change.